src/scenes/AcceleratorRouter/People/NewView.tsx

```diff
@@ -61,6 +61,12 @@ const NewView = () => {
       return;
     }
 
+    // Email address must end in @terraformation.com
+    if (!/@terraformation.com$/.exec(debouncedEmail)) {
```

Comment:
another option if you prefer is `debouncedEmail.endsWith('@terraformation.com')`